### PR TITLE
make-debs.sh: clean dir to allow building deb packages multiple times

### PR DESCRIPTION
**Pull request by iotcg**

### CODE REVIEW

File: make-debs.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ git clean -dxf
 # c) compares higher than any previous commit
 # d) contains the short hash of the commit
 #
-vers=$(git describe --match "v*" | sed s/^v//)
+vers=$(git describe --match "v*" | sed 's/^v//')
```
Comment (tchaikov) [inline make-debs.sh]: why do you think it was wrong without the quotes? could you update the commit message with the rationale?

File: make-debs.sh [hunk 2]
```diff
@@ -19,7 +19,7 @@ set -xe
 base=${1:-/tmp/release}
 codename=$(lsb_release -sc)
 releasedir=$base/$(lsb_release -si)/WORKDIR
-rm -fr $releasedir
+rm -fr $base/(lsb_release -si)
```
Comment (tchaikov) [inline make-debs.sh]: could you elaborate a little bit what "2nd time build error" stands for?

Comment (iotcg) [reply make-debs.sh]: @tchaikov The error is below:
_File "pool/main/c/ceph/rbd-nbd_14.0.1-927-g004d918ae0-1_amd64.deb" is already registered with different checksums!
md5 expected: bb874edadc09b586d8947780ccf32757, got: cfc00c331cdb81ffe9732883355a99dd
sha1 expected: 59f6e5c1905b05369f0d7780fc1ed5aa1d9ea0f4, got: 36bf0b427dd4b29625cb5b19e874cfa61688cb02
sha256 expected: 2e0f7486e0ae379fe16b3efc414a0677e264b13af12070dd4712b32eca94b042, got: 9bd22cfb676572afa643f20ba1a2d50489a312faf5065322ae3638db9e2cded8
size expected: 121452, got: 121496
There have been errors!_

Reproduce steps:
1. do_cmake.sh
2. cd build; make; cd -
3. make-debs.sh #no error
4. change source code and build again "cd build; make; cd -"
5. make-debs.sh #this above error

File: make-debs.sh (hunk 1 above)
Comment (iotcg) [reply make-debs.sh]: @tchaikov Please ignore this commit.
I hits the sed usage error 3 weeks ago when building ceph ubuntu packages. Now there's no error without this commit.

**Comment by iotcg** [comment]
@tchaikov Do we need to resolve the "2nd time build error"? If yes, I'll re-submit this PR. Or, I'll close this PR.

File: make-debs.sh (hunk 2 above)
Comment (tchaikov) [inline make-debs.sh]: i see... probably we can something like
```sh
rm -fr $(dirname $releasedir)
```
less repeating this way.

**Comment by tchaikov** [comment]
@iotcg seems this script is not referenced anywhere in Ceph project. but it's been updated. so i think your fix will also help others who want to run this script multiple times without cleaning the built deb packages.

so, could you remove the sed change. and repush?

thanks!

**Comment by iotcg** [comment]
@tchaikov Thanks. Hope it can help others.

**Comment by tchaikov** [comment]
@iotcg could you remove the `Reviewed-by: ` line in your commit message? we will add it in the merge commit.

**Comment by iotcg** [comment]
@tchaikov Thanks for your notification.

**Comment by iotcg** [comment]
Jenkins, test this please

**Comment by iotcg** [comment]
@tchaikov Jenkins build can't pass test. It's not patch's problem.
The build log shows:
https://jenkins.ceph.com/job/ceph-pull-requests-arm64/26249/console
stderr: fatal: unable to access 'https://github.com/ceph/ceph.git/': Could not resolve host: github.com

**Comment by tchaikov** [comment]
Le sam. 24 nov. 2018 à 14:08, Changcheng <notifications@github.com> a
écrit :

> @tchaikov <https://github.com/tchaikov> Jenkins build can't pass test.
> It's not patch's problem.
> The build log shows:
> https://jenkins.ceph.com/job/ceph-pull-requests-arm64/26249/console
> stderr: fatal: unable to access 'https://github.com/ceph/ceph.git/':
> Could not resolve host: github.com
>

yeah, it’s a known issue. but it’s a long weekend. probably we will get it
fixed early next week. thanks for looking into it=)

have a nice weekend!

<http://github.com>

**Comment by iotcg** [comment]
Jenkins, test this please

**Pull request merged**